new_tab/about.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="../css_folder/style.css" />
+ </head>
+ <body>
+  <header>
+   <nav class="header">
+    <div class="navbarMobile">
+     <ul class="menumo">
+      <li class="grid">
+       <a href="#"><ion-icon name="grid"></ion-icon></a>
+      </li>
+      <li class="menu-nth"></li>
+      <li class="subscribe">Subscribe</li>
+     </ul>
+    </div>
+   </nav>
+
+   <nav class="header1">
+    <ul>
+     <li><ion-icon name="logo-facebook"></ion-icon></li>
+     <li><ion-icon name="logo-twitter"></ion-icon></li>
+     <li>English</li>
+     <li>My Page</li>
+     <li>Logout</li>
+    </ul>
+   </nav>
+   <nav class="header2">
+    <div class="navbarDesktop">
+     <span
+      >GLCon23'<ion-icon id="naveicon" name="aperture-outline"></ion-icon
+     ></span>
+     <ul>

```

Comment:
- [ ] please, like the previous reviewer noted, kindly change the background of the mobile headline section. Please use a clearer background. Remember, the design should look alike, and only the contents should be personalized.below you can see how different it looks from the original design
yours
![Screenshot (1225)](https://github.com/Mike47ip/JavascriptCapstone/assets/65298048/17a27bd9-372b-4333-9e79-a6262e9693b8)
original
![Screenshot (1034)](https://github.com/Mike47ip/JavascriptCapstone/assets/65298048/568d692c-a6b5-4904-a230-c3c13457c247)